plugin-server/src/worker/ingestion/process-event.ts

```diff
@@ -209,7 +209,32 @@ export class EventsProcessor {
         return now
     }
 
+    public isNewPersonPropertiesUpdateEnabled(teamId: number): boolean {
+        try {
+            const teamsStrs = this.pluginsServer.NEW_PERSON_PROPERTIES_UPDATE_ENABLED_TEAMS.split(',').filter(String)
+            const teams = teamsStrs.map((teamId) => Number(teamId))
```

Comment:
nit: actually (and I think I made this "mistake" recently too somewhere), this can just be:

```suggestion
            const teams = teamsStrs.map(Number)
```